webapp/models.py

```diff
@@ -165,6 +173,7 @@ def _parse_document_topic(self, topic):
                 updated_datetime.replace(tzinfo=None)
             ),
             "forum_link": f"{self.base_url}/t/{topic['slug']}/{topic['id']}",
```

Comment:
Maybe it could be remove here to avoid having redirections all the times

Reply:
What do you mean? The forum link will still have the `/t/`. Discourse hasn't changed.

Reply:
as replied before, this me being bad